src/components/Coverage/Coverage.js

```diff
@@ -56,33 +57,43 @@ export default class Coverage extends React.Component {
     const { pci, line, eResource } = this.props;
     let isMonograph = false;
     let dataToRender;
+    let embargo;
```

Comment:
Because of the various ways coverage is stored, this is unfortunately a pretty complicated component. Because of that the prop types are actually pretty valuable here in helping us understand what the expected shape is of the various props that can be passed in. So can you extend the prop types for this component to include where the embargo info is for the three ways this component can be called? And maybe include the `titleInstance.type.value` structure in it as well?

Because even after looking at the backend implementation and JIRA issues for this, I _still_ struggle to follow this and can't for sure tell whether everything's fine or something's being missed here 😆 (eg, sometimes we use `dataToRender.embargo` even after we save it off to `embargo`).

Reply:
I've already had a conversation with Aditya about that. Because of my extreme laziness (home office is not good for me) I've tried to avoid extending the prop types. I should have known that I couldn't get away with it ...

Reply:
Whenever we can make the code easier to understand it helps the rest of us who have the same problem right now. 🙃